helix-term/src/ui/prompt.rs

```diff
@@ -56,6 +56,10 @@ pub enum Movement {
     None,
 }
 
+fn is_word_sep(c: char) -> bool {
+    c == '/' || c.is_whitespace()
```

Comment:
should `\` be included for windows?

Reply:
You want to use std::path::MAIN_SEPARATOR which will be different depending on platform